08-CMSIS/core/src/threads.c

```diff
@@ -49,22 +49,20 @@ void __attribute__((naked)) pendsv_handler()
 void thread_start()
 {
 	lastTask = 0;
+	CONTROL_Type PSP_in_Thread_mode;
+	PSP_in_Thread_mode.b.nPRIV = 1;
+	PSP_in_Thread_mode.b.SPSEL = 1;
 
```

Comment:
Rewrite as following:
```
CONTROL_Type PSP_in_Thread_mode = { .b.nPRIV = 1, .b.SPSEL = 1 };
```
However, the naming `PSP_in_Thread_mode` breaks the simplicity.